index.js

```diff
@@ -1,31 +1,147 @@
+    featured_image: 'images/Snapshoot Portfolio_1.png',
+    technologies: ['Ruby on rails', 'css', 'JavaScript', 'html'],
+    link_to_live: 'https://ibrahim777764.github.io/new_portfolio/',
+    link_to_source: 'https://github.com/ibrahim777764/new_portfolio',
+  },
+  {
+    name: 'Multi-Post Stories Gain+Glory',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea',
+    featured_image: 'images/Snapshoot Portfolio_1.png',
+    technologies: ['Ruby on rails', 'css', 'JavaScript', 'html'],
+    link_to_live: 'https://ibrahim777764.github.io/new_portfolio/',
+    link_to_source: 'https://github.com/ibrahim777764/new_portfolio',
+  },
+  {
+    name: 'Multi-Post Stories Gain+Glory',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea',
+    featured_image: 'images/Snapshoot Portfolio_1.png',
+    technologies: ['Ruby on rails', 'css', 'JavaScript', 'html'],
+    link_to_live: 'https://ibrahim777764.github.io/new_portfolio/',
+    link_to_source: 'https://github.com/ibrahim777764/new_portfolio/',
+  },
+  ];
+
+  const gridTemplate = document.getElementById('grid-template');
+  const recentWorks = document.getElementById('Recent-id');
+  const detailsPopUpHeading = document.querySelector('.detail-popup-heading');
+  const popupWindow = document.getElementById('mobile-details-modal');
+  projects.forEach((project) => {
+    const clonedNode = gridTemplate.children[0].cloneNode(true);
+    clonedNode.children[0].children[0].innerText = project.name;
+    recentWorks.appendChild(clonedNode);
+
+    detailsPopUpHeading.textContent = project.name;
+
+    const btnPopups = recentWorks.querySelectorAll('.btn-popup');
+    btnPopups.forEach((button) => {
+      button.addEventListener('click', () => {
+        popupWindow.classList.remove('hidden');
+      });
+    });
+  });
```

Comment:
Kindly try to match the `see live` and `see source` in the desktop version width `992px` to match the Figma design. 
| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/157516392-9a86fe29-dc76-4731-8790-ec74ec576b2a.png) | ![image](https://user-images.githubusercontent.com/24703171/157516259-73649f7e-b706-4227-86f9-d413b3d9d195.png) |
the text and icon should be beside each other.